net.ptilouk.superfluous.yml

```diff
@@ -0,0 +1,117 @@
+app-id: net.ptilouk.superfluous
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: superfluous-returnz
+rename-icon: superfluous-returnz
+rename-desktop-file: superfluous-returnz.desktop
+rename-appdata-file: superfluous-returnz.metainfo.xml
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=all
```

Comment:
why does it need all?

Reply:
This was a mistake (I took inspiration on other games on Flathub, but actually I just need dri), I'll change it.